src/app/messages-page/messages-inbox/messages-inbox.component.ts

```diff
@@ -51,6 +82,190 @@ export class MessagesInboxComponent implements OnInit, OnChanges {
     }
   }
 
+  showMoreButton() {
+    return !(this.newMessagesFromPage != null && this.newMessagesFromPage < this.globalVars.messagesPerFetch);
```

Comment:
could be rewritten as `!this.newMessagesFromPage || this.newMessagesFromPage >= this.globalVars.messagesPerFetch;`